com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1066,8 +1066,7 @@ public bool StartHost()
 
             if (NetworkConfig.ConnectionApproval && ConnectionApprovalCallback != null)
             {
-                var response = new ConnectionApprovalResponse();
```

Comment:
We actually want and need the `ConnectionApprovalResponse` to  be allocated in `NetworkManager` and passed to client code. The reason for this is that the connection approval now has a `Pending` field and this pending field can be used by client code to indicate when the response is filled at a later time.

Reply:
So, is that a '*ConnectionApprovalCallback needs to be a `Func<ConnectionApprovalReqest, ConnectionApprovalResponse, ConnectionApprovalResponse>` so the response can be passed to the client code and returned back*', or is that a '*ConnectionApprovalCallback should remain as an `Action<ConnectionApprovalRequest, ConnectionApprovalResponse>`*'?

If it's the former, I can go fix that, but if it's the latter, you might want to update your release notes for pre-10 to remove the thing saying that `ConnectionApprovalCallback` is supposed to be a `Func<>`.